modules/test/playwright/tests/osb-faro-web/utils/navigation.ts

```diff
@@ -49,3 +49,17 @@ export async function navigateToACSitesPageViaURL({
 		`${faroConfig.environment.baseUrl}/workspace/${projectID}/${channelID}/sites`
 	);
 }
+
+export async function navigateToACTestPageViaURL({
+	channelID,
+	page,
+	projectID,
+}: {
+	channelID: string;
+	page: Page;
+	projectID: string;
+}) {
+	await page.goto(
+		`${faroConfig.environment.baseUrl}/workspace/${projectID}/${channelID}/tests`
+	);
+}
```

Comment:
Hey @marcosapmf, I think creating a function to navigate to each page URL in AC won't be very good. I think we'll have several similar functions just for navigation. I think we could follow one of these suggestions:

1. Create an `enum` with the final term of the URL that represents each page, then we could use the `enum` values ​​in a single function and the function would receive as a parameter the value of the `enum` that would be used:
```
export enum ACPages {
	sitePage = 'sites',
	assetPage = 'assets',
	eventAnalysisPage = 'events',
	segmentPage = 'segments',
	individualPage = 'individuals',
	testPage = 'tests',
}

export async function navigateToACPageViaURL({
	ACPage,
	channelID,
	page,
	projectID,
}: {
	ACPage: ACPages;
	channelID: string;
	page: Page;
	projectID: string;
}) {
	await page.goto(
		`${faroConfig.environment.baseUrl}/workspace/${projectID}/${channelID}/${ACPage}`
	);
}
```

2. We can just use `navigateToACSitesPageViaURL` to access AC via URL and then navigate in the UI to the pages we want to access in sequence using the `navigateTo` function.

What do you think?

Reply:
Agreed @ViniLopes20. Let's follow your first suggestion :)